### PR TITLE
Template

### PR DESCRIPTION
**Pull request by dafquers**
J'ai rajouté un programme qui permet de rajouter des livres et de le mettre dans l'inventaire

### CODE REVIEW

File: src/db/mongoConnector.js [hunk 1]
```diff
@@ -174,12 +172,14 @@ class MongoConnector {
...
    * @output new inventory in database
    */
-  async createInventory(data) {
+  async createInventory(data, type = "custom") {
     let document = data.datas;
     document["col_name"] = true; //(true) if it is the object with the names of the columns, (false) if an item of the inventory
     document["desc"] = data.desc;
     document["name-list"] = data.namelist;
+    document["type"] = type;
     document["sharing"] = data.sharing;
```
Comment (TheoTechnicguy) [inline src/db/mongoConnector.js:181]: Suggestion: where available, use `document.<attribute> = `

File: src/routes/inventory-books.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+const books = require("./template-books");
+const { route } = require("./public");
+
+const router = express.Router();
+
+/**
+ * Search books.
+ *
+ * @path /insert_inventory_books
+ * @method POST
```
Comment (TheoTechnicguy) [inline src/routes/inventory-books.js:20]: > Note: you describe a `POST` request, where you only have a `GET`

File: src/routes/inventory-books.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    await db.insertItemInventory(data);
+    res.redirect(`/show_inventory?title=${data.coll}`);
+  }
+})
+
+/**
+ * Create of inventory of type books.
+ *
+ * @path /add_books
+ * @method Get
```
Comment (TheoTechnicguy) [inline src/routes/inventory-books.js:65]: Please be consistent: `GET`
```suggestion
 * @method GET
```

**Comment by TheoTechnicguy** [comment]
Some minor suggestions.
The code seems good, but be careful to follow the guidelines and think about prettification - it's not a big thing but it makes the code consistent.

**Comment by Sorio5** [comment]
I've fixed some conflicts, should be okay, don't hesitate to check or give your reviews.

**Review by TheoTechnicguy — APPROVED** [review]

**Comment by TheoTechnicguy** [comment]
Do watch out and **extensively** test!

**Pull request merged**